src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import './utils/variables';
```

Comment:
Ensure that the `variables` file contains all the required SCSS variables like `$main-color`, `$main-during`, `$card-width`, `$main_accent-color`, `$secondary-color`, and `$blue_accent-color`. These variables are crucial for the styling and functionality of your components.